### PR TITLE
Add v1.4.0 release blog post

### PR DESCRIPTION
**Pull request by pmuens**
Adds the blog post for the Release of Serverless v1.4.0.

### CODE REVIEW

File: posts/2016-12-15-serverless-v1.4.0.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+---
+title: C# support, CLI based plugin discovery and management, Alexa event support, service wide metrics in Serverless Framework v1.4
+description: C# support, CLI based plugin discovery and management, Alexa event support, service wide metrics in Serverless Framework v1.4
+date: 2016-12-15
+layout: Post
+---
+
+We're happ to release v1.4.0 of the Serverless Framework today!
```
Comment (mthenw) [inline posts/2016-12-15-serverless-v1.4.0.md]: happy

File: posts/2016-12-15-serverless-v1.4.0.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
+---
+title: C# support, CLI based plugin discovery and management, Alexa event support, service wide metrics in Serverless Framework v1.4
```
Comment (mthenw) [inline posts/2016-12-15-serverless-v1.4.0.md]: Maybe "C# template,..."

File: posts/2016-12-15-serverless-v1.4.0.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+      - alexaSkill: true
+```
+
+You can read more in [the docs](https://serverless.com/framework/docs/providers/aws/events/alexa-skill) about this new event source.
+
+### C# service template
+
+AWS introduced Dotnet core support for Lambdas at re:Invent.
+
+Serverless has you covered and ships with the new `aws-csharp` template you can use to create C# services on AWS Lambda.
```
Comment (mthenw) [inline posts/2016-12-15-serverless-v1.4.0.md]: we should add link to https://github.com/serverless/serverless/tree/master/docs/providers/aws/examples/hello-world/csharp somwhere here. C# requires some dotnet tools to run before `sls deploy`

File: posts/2016-12-15-serverless-v1.4.0.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+v1.4 now supports stdin so that you can pipe data directly into the `serverless invoke` call like this:
+
+```bash
+echo "hello world!" | serverless invoke --function hello --stage dev --region us-east-1
+```
+
+This gives you way more flexibility to invoke a function with different data. Here's an example how a `dataGenerator.js` file could be used to generate data for your function:
+
+```bash
+dataGenerator.js | serverless invoke local --function hello
```
Comment (mthenw) [inline posts/2016-12-15-serverless-v1.4.0.md]: `node dataGenerator.js | serverless invoke local --function hello` ?

Comment (pmuens) [inline posts/2016-12-15-serverless-v1.4.0.md]: Good find. This should be updated as well:

https://github.com/serverless/serverless/blob/master/docs/providers/aws/cli-reference/invoke.md#function-invocation-with-data-from-standard-input

https://github.com/serverless/serverless/blob/master/docs/providers/aws/cli-reference/invoke.md#function-invocation-with-data-from-standard-input

File: posts/2016-12-15-serverless-v1.4.0.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+```bash
+serverless create --template aws-csharp
+```
+
+to create a new C# service.
+
+The [C# "hello world" example](https://serverless.com/framework/docs/providers/aws/examples/hello-world/csharp/) will show you how you can setup and deploy your C# service.
+
+**Note:** Serverless supports all [AWS Lambda runtimes](http://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html) out of the box. You can change the runtime with the help of the `runtime` property in [`serverless.yml` file](https://serverless.com/framework/docs/providers/aws/guide/serverless.yml/).
```
Comment (mthenw) [inline posts/2016-12-15-serverless-v1.4.0.md]: Thanks for that!

File: posts/2016-12-15-serverless-v1.4.0.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+### Directory support for service installation via git
```
Comment (mthenw) [inline posts/2016-12-15-serverless-v1.4.0.md]: well, not exactly git. We support only github, right?

Comment (pmuens) [inline posts/2016-12-15-serverless-v1.4.0.md]: Nice find! Will update it 👍

**Pull request merged**